src/prop/theory_proxy.cpp

```diff
@@ -156,16 +156,14 @@ void TheoryProxy::spendResource(ResourceManager::Resource r)
   d_theoryEngine->spendResource(r);
 }
 
-bool TheoryProxy::isDecisionRelevant(SatVariable var) {
-  return d_decisionEngine->isRelevant(var);
-}
+bool TheoryProxy::isDecisionRelevant(SatVariable var) { return true; }
```

Comment:
This and the change below look a bit odd. :) Could this affect `decision==internal`? Moreover, could you comment on these changes and update the doc (currently non-existent hehe) in the headers? OTOH, if these methods are to behave like this then we should change the SAT solver to not rely on them as well.

Reply:
This change preserves behavior, isRelevant always returned true.  We could do the further refactoring to not make SAT solver rely on this method. I considered doing this on this PR but decided to wait for now.

Reply:
OK, sounds good.